core/schemas/entity.py

```diff
@@ -130,10 +130,29 @@ class Investigation(Entity):
     reference: str = ""
 
 
+class SeverityType(str, Enum):
+    none = "None"
+    low = "Low"
+    medium = "Medium"
+    high = "High"
+    critical = "Critical"
```

Comment:
let's make these lowercase

```suggestion
    none = "none"
    low = "low"
    medium = "medium"
    high = "high"
    critical = "critical"
```